catroid/src/debug/AndroidManifest.xml

```diff
@@ -35,13 +35,20 @@
         <meta-data android:name="com.crashlytics.ApiKey" android:value="YOUR_API_KEY"/>
         <activity
             android:name=".uiespresso.ui.activity.ProjectUploadDialogTest$ProjectUploadTestActivity"
+            android:exported="true"
             android:screenOrientation="portrait"
             android:windowSoftInputMode="adjustNothing">
             <intent-filter android:label="@string/app_name">
                 <action android:name="android.intent.action.VIEW"/>
             </intent-filter>
         </activity>
 
+        <activity
```

Comment:
Why do we need this for debugging now? Just curious :)

Reply:
For each activity we need to set this flag to true, if it contains an intent filter. And this flag allows it, that the activity can be launched from an external application and this makes sense for activities with intent filters since they can respond to a specific action triggered by an external application.